### PR TITLE
Updates to data sets

### LINKED ISSUE #43: Add a one line per patient data set
Add a data set to pkg that is one line per patient, so we can add example using `tbl_adverse_event(id_df=)`

### LINKED ISSUE #44: Add more cols to `df_adverse_events`?
Add these columns to the data set for short examples?

```r
  df_adverse_events %>%
  dplyr::mutate(
    any_complication = TRUE,
    grade3_complication = grade >= 3
  )
```

### PR DESCRIPTION
**Pull request by ddsjoberg**
- Added a new data set that is one line per patient
- Added 2 columns to `df_adverse_events` to use as example in the `tbl_ae_focus()` function
- Will need to update examples in `tbl_ae_focus()` once the data and function live on the same branch

closes #44 
closes #43

### CODE REVIEW

File: data-raw/DATASET.R [hunk 1]
```diff
@@ -9,7 +9,7 @@ df_adverse_events <-
                "Iron deficiency anaemia", "Thrombocytopenia"),
              size = n, replace = TRUE),
     grade = sample.int(5, n, replace = TRUE)
-  ) %>%
+  ) |>
```
Comment (shannonpileggi) [inline data-raw/DATASET.R:12]: Do you want to use base R pipe instead of magrittr pipe throughout package?

File: data-raw/DATASET.R [hunk 2]
```diff
@@ -20,32 +20,56 @@ df_adverse_events <-
...
+    any_complication = "Any Grade Complication",
+    grade3_complication = "Grade 3+ Complication"
   )
 
-usethis::use_data(df_adverse_events, overwrite = TRUE)
+df_patient_characteristics <-
+  tibble::tibble(
+    patient_id = paste("ID", 1:100),
+    trt = sample(c("Drug A", "Drug B"), size = 100, replace = TRUE),
+    age = rnorm(100, 50, 10)
```
Comment (shannonpileggi) [inline data-raw/DATASET.R:57]: Suggesting whole numbers on age.

```suggestion
    age = floor(rnorm(100, 50, 10))
```

File: tests/testthat/test-add_overall.R [hunk 3]
```diff
@@ -7,17 +7,32 @@ test_that("multiplication works", {
         ae = adverse_event,
```
Comment (shannonpileggi) [inline tests/testthat/test-add_overall.R:7]: This comment isn't in the right place, sorry.  Wasn't letting me review line 1.

```suggestion
        test_that("add_overall() works", {
```

**Review by shannonpileggi — CHANGES_REQUESTED** [review]
Thanks for updating the data, this looks great!

File: data-raw/DATASET.R (hunk 1 above)
Comment (ddsjoberg) [reply data-raw/DATASET.R:12]: I think for now it's better to use `%>%` unless we want to require R >= 4.1

Comment (ddsjoberg) [reply data-raw/DATASET.R:12]: Oh, just using this here in this script that is not included in the package builds. this makes this script stand-alone and doesn't require loading of any other packages.

File: tests/testthat/test-add_overall.R (hunk 3 above)
Comment (ddsjoberg) [reply tests/testthat/test-add_overall.R:7]: just pushed the updated comment!

**Review by shannonpileggi — APPROVED** [review]
Amazing!

**Comment by ddsjoberg** [comment]
YAY! I just pulled the updates from the main branch and will merge when all the CI checks pass

**Pull request merged**